doc/manual/mathematical-operations.rst

```diff
@@ -454,6 +454,7 @@ Function                     Description
 :func:`cld(x,y) <cld>`       ceiling division; quotient rounded towards ``+Inf``
 :func:`rem(x,y) <rem>`       remainder; satisfies ``x == div(x,y)*y + rem(x,y)``; sign matches ``x``
 :func:`mod(x,y) <mod>`       modulus; satisfies ``x == fld(x,y)*y + mod(x,y)``; sign matches ``y``
+:func:`mod1(x,y) <mod1>`     modulus with offset 1; gives a result ``r`` such that ``mod(r, y)==mod(x,y)``; sign matches ``y``
```

Comment:
I found this definition in Mathematica's docs. Very few other programming languages seem to have mod with offset.